modular-docs-manual/content/topics/introduction.adoc

```diff
@@ -2,3 +2,5 @@
 = Introduction
 
 This manual provides instructions on how to author modularly structured documentation based on user stories. The manual defines used terminology, describes components that form modular documentation, and instructs writers on how to use provided templates to turn user stories into modular documentation.
+
+IMPORTANT: As CCS strategic imperatives and Pantheon 2 development continue to evolve, the details and recommendations in this guide might be modified. The Flexible Customer Content (FCC) team recognizes that the fluctuation in recommendations can be frustrating for CCS teams trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for CCS documentation.
```

Comment:
Admonitions take the following format:

```
[IMPORTANT]
====
Read this note.
====
```

Reply:
@oraNod That's just for mult-line block admonitions. For single-line admonitions, both formats render the same, so it's a matter of preference, imo. I prefer using this format for single-line admonitions because it's only one ascii line, faster, more streamlined. I reserve the other format for multi-line block notes, which is I believe why that format exists anyway: https://asciidoctor.org/docs/user-manual/#admonition

Reply:
Fair enough. Just a review reflex on my part. I typically enforce the multi-line format throughout docs just to make it easy and so we're always consistent.